core/src/modules/ops/source.ts

```diff
@@ -262,6 +262,21 @@ export namespace SourceOps {
       profileProperties,
     };
 
+    const { pluginConnection } = await source.getPlugin();
+    if (!pluginConnection) {
+      throw new Error(
+        `cannot find connection for source ${source.type} (${source.id})`
+      );
+    }
+
+    const canImport = pluginConnection.methods.profileProperty;
```

Comment:
Is there ever a case when a plugin connection has `methods.profileProperties` and not `methods.profileProperty`?

Reply:
Our plugins implement both, but yeah, it's possible. However, at least at the moment the synchronous profile importing (import and export profile button) exclusively uses the single `profileProperty` method.